src/charm.py

```diff
@@ -744,12 +935,15 @@ def push_tls_certificate_to_workload(self) -> None:
 
     def delete_tls_certificate_from_workload(self) -> None:
         """Deletes certificate from the workload container."""
-        logger.error("Deleting TLS certificate from workload container")
+        logger.info("Deleting TLS certificate from workload container")
```

Comment:
Changing this back to error (just don't wanna interfere with ongoing pipelines.)

Reply:
I think "info" is more relevant

Reply:
Not changing it then :-)